src/GlobalStyles.js

```diff
@@ -2,20 +2,28 @@ import { createGlobalStyle } from 'styled-components/macro'
 
 export default createGlobalStyle`
 body {
-   font-family: sans-serif;
+   box-sizing: border-box;
+   margin: 20px;
 }
 
-h2 {
-   margin-top: 10px;
-   font-size: 16px;
-   position: relative;
+h1 {
+   font-family: Helvetica;
```

Comment:
Do you use other font-familys as well? Otherwise I would move this to the universal selector *.